### PR TITLE
Added Depth's Swag

### LINKED ISSUE #1077: [Hacktoberfest] Add Swag Opportunity for Depth's 30 Days Open Source Learning Challenge
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. :   <a href=https://www.depths.so/>Depths</a>

Q. Where have you seen the announcement (URL)?  
A. :  https://www.depths.so/30daysopensource

Q. What kind of swag (clothing, stickers, ...)?  
A. :  Clothing,Stickers....

Q. Have you received swag from this?  
A. :  No

Q. How long did it take?  
A. :  N/A

Q. Does it ship worldwide?  
A. :  Yes

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by BandhiyaHardik**
Add Swag Opportunity for Depths' 30 Days Open Source Learning Challenge

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I've added an opportunity to get swags from Depths by Completing the #30DaysOpenSource and including a Hacktoberfest merged PR. This fixes #1077

### CODE REVIEW

**Comment by aslafy-z** [comment]
Please rebase your pull request!

**Comment by BandhiyaHardik** [comment]
@aslafy-z    I've allowed rebasing. 

can you add hacktoberfest label?

File: data.json [hunk 1]
```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2020-10-01T19:00:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Depths",
+    "difficulty": "Medium",
```
Comment (aslafy-z) [inline data.json:211]: Please fix this line, see the build log at https://github.com/swapagarwal/swag-for-dev/actions/runs/6485621033/job/17613034556?pr=1079

Comment (BandhiyaHardik) [reply data.json:211]: okay i will do it now.

**Comment by aslafy-z** [comment]
@BandhiyaHardik  The hacktoberfest label is added at the repository level, there is no need for it to be added to the pull request.

File: data.json [hunk 2]
```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2020-10-01T19:00:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Depths",
+    "difficulty": "medium",
+    "description": "Read 2 curated learning resources daily, share insights in comments, and complete 30 days of challenges for rewards, including a Hacktoberfest merged PR.",
```
Comment (aslafy-z) [inline data.json:212]:
```suggestion
    "description": "Read 2 curated learning resources daily, share insights in comments, complete 30 days of challenges for rewards and a Hacktoberfest merged PR, get a free Depths swag!",
```

**Review by aslafy-z — APPROVED** [review]

**Pull request merged**